rpm/kubelet.spec

```diff
@@ -1,5 +1,5 @@
 %global KUBE_VERSION 1.4.4
-%global KUBEADM_VERSION 1.5.0-alpha.1.409+714f816a349e79
+%global KUBEADM_VERSION v1.5.0-alpha.1.955+b1d8961fe43015
```

Comment:
Added "v" here looks wrong, will end up with a bad URL when fetching as it's still there as well.